src/locales/en.json

```diff
@@ -222,4 +223,4 @@
     "copy_link": "Copy link to this scenario",
     "reset_form": "Reset form"
   }
-}
+}
```

Comment:
Why does your editor keep removing newlines? Please put it back 😛

Reply:
Haha. Good question. I'll find a setting for that. I guess our linter doesn't run on JSON files???

Reply:
K I found the setting. Won't happen again. Good catch.